packages/mds-compliance-service/repository/migrations/1632935069192-AddComplianceViolations.ts

```diff
@@ -5,7 +5,7 @@ export class AddComplianceViolations1632935069192 implements MigrationInterface
 
   public async up(queryRunner: QueryRunner): Promise<void> {
     await queryRunner.query(
-      `CREATE TABLE "compliance_violations" ("recorded" bigint NOT NULL DEFAULT (extract(epoch from now()) * 1000)::bigint, "id" BIGSERIAL NOT NULL, "violation_id" uuid NOT NULL, "timestamp" bigint NOT NULL, "policy_id" uuid NOT NULL, "provider_id" uuid NOT NULL, "rule_id" uuid NOT NULL, "event_timestamp" bigint NOT NULL, "device_id" uuid NOT NULL, "trip_id" uuid, CONSTRAINT "compliance_violations_pkey" PRIMARY KEY ("violation_id"))`
+      `CREATE TABLE "compliance_violations" ("recorded" bigint NOT NULL DEFAULT (extract(epoch from now()) * 1000)::bigint, "id" BIGINT GENERATED ALWAYS AS IDENTITY, "violation_id" uuid NOT NULL, "timestamp" bigint NOT NULL, "policy_id" uuid NOT NULL, "provider_id" uuid NOT NULL, "rule_id" uuid NOT NULL, "event_timestamp" bigint NOT NULL, "device_id" uuid NOT NULL, "trip_id" uuid, CONSTRAINT "compliance_violations_pkey" PRIMARY KEY ("violation_id"))`
```

Comment:
TypeORM doesn't know about `IDENTITY` columns, but we use them over `BIGSERIAL`. We always have to make this change manually. Again, not this was an existing table. BTW, I renamed this file to remove the `_`s and follow existing conventions.